pkg/funky/response.go

```diff
@@ -0,0 +1,33 @@
+package funky
+
+// Constants indicating the types of errors for Dispatch function invocation
+const (
+	InputError    = "InputError"
+	FunctionError = "FunctionError"
+	SystemError   = "SystemError"
+)
+
+// Response a struct to hold the response to a Dispatch function invocation
+type Response struct {
+	Context Context     `json:"context"`
+	Payload interface{} `json:"payload"`
+}
+
+// Context a struct to hold the context of a Dispatch function invocation
+type Context struct {
+	Error Error `json:"error"`
```

Comment:
Should be of type `*Error`: might not be present or needed (e.g. as part of request message)